renovate.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
I was going to comment that we probably also had to disable the renovate app in the repository, but I just discovered that we simply never enabled it after moving repositories so this didn't actually do anything 😅